README.md

```diff
@@ -236,7 +236,7 @@ func scrollViewsToTest(in view: V, identifier: String) -> [String: UIScrollView]
 
 
 In case you have to wait for asynchronous operations before rendering the UI and take the screenshot, you can leverage the `isViewReady(view:identifier:)` method.
-For instance, here we wait until an hypotetical view that shows an image from a remote URL is ready. When the image is shown (that is, the state is `loaded`, then the snapshot is taken)
+For instance, here we wait until an hypothetical view that shows an image from a remote URL is ready. When the image is shown (that is, the state is `loaded`, then the snapshot is taken)
 ```swift
 import TempuraTesting
 
```

Comment:
I would add to the readme a section about the keyboard testing. Something along the lines you specified in the description of the PR.